swagger.yaml

```
@@ -3445,19 +3445,60 @@ paths:
           description: The cluster to return events for.
           type: string
           format: uuid
-          required: False
+          required: false
         - in: query
           name: host_id
-          description: A host in the specified cluster to return events for.
+          description: A host in the specified cluster to return events for (DEPRECATED).
```

let's follow the existing conventions:
```suggestion
          description: (DEPRECATED) A host in the specified cluster to return events for.
```